### PR TITLE
Update expense, purchase and github issues

### PR DESCRIPTION
**Pull request by emran-wedevs**

### CODE REVIEW

File: includes/updates/update-1.6.7.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+namespace WeDevs\ERP\HRM\Update;
+
+/*
+ * Add transaction_charge column in `erp_acct_expenses` table
+ */
+function erp_acct_alter_acct_expenses_1_6_7() {
+    global $wpdb;
+
+    $table = $wpdb->prefix . 'erp_acct_expenses';
+    $cols  = $wpdb->get_col( $wpdb->prepare( "DESC $table" ) );
```
Comment (OpuRockey) [inline includes/updates/update-1.6.7.php:11]: Use prepare statement with proper placeholder.

File: includes/updates/update-1.6.7.php [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+/*
+ *  Add transaction_charge and ref column in `erp_acct_pay_purchase` table
+ */
+function erp_acct_alter_pay_purchase_1_6_7() {
+    global $wpdb;
+
+    $table = $wpdb->prefix . 'erp_acct_pay_purchase';
+    $cols  = $wpdb->get_col( $wpdb->prepare( "DESC $table" ) );
```
Comment (OpuRockey) [inline includes/updates/update-1.6.7.php:29]: Please use prepare statement properly.

File: modules/accounting/includes/functions/product-cats.php [hunk 3]
```diff
@@ -52,14 +52,14 @@ function erp_acct_insert_product_cat( $data ) {
...
             $wpdb->prefix . 'erp_acct_product_categories',
-            [
-                'name'       => $product_cat_data['name'],
-                'parent'     => isset($product_cat_data['parent']['id']) ? $product_cat_data['parent']['id'] : 0,
-                'created_at' => $product_cat_data['created_at'],
-                'created_by' => $product_cat_data['created_by'],
-                'updated_at' => $product_cat_data['updated_at'],
-                'updated_by' => $product_cat_data['updated_by'],
-            ]
+            array(
```
Comment (OpuRockey) [inline modules/accounting/includes/functions/product-cats.php:55]: Did you run phpcbf for this file?

Comment (emran-wedevs) [reply modules/accounting/includes/functions/product-cats.php:55]: no

**Comment by nurul-umbhiya** [comment]
@emran-wedevs you need to add all db related changes to **includes/class-install.php** also include **update-1.6.7.php** file to **includes/class-updates.php**

**Pull request merged**